src/lit-html.ts

```diff
@@ -268,23 +268,34 @@ export class NodePart extends Part {
   setValue(value: any): void {
     value = this._getValue(value);
 
-    if (value instanceof Node) {
-      this._previousValue = this._setNode(value);
+    if (value === undefined ||
```

Comment:
Yes, this is trying to compare primitives only. I'll try that above.

@TimvdLippe that function is odd, it's using `===` compares even on objects